packages/awslint/lib/case.ts

```diff
@@ -0,0 +1,18 @@
+import * as changeCase from 'change-case';
```

Comment:
I tested a bunch. 

`fast-case` is the fastest, but cannot deal with cases like `DBProxy` (converts it to `dBProxy`)
`lodash` and `change-case` are of similar speed
`camelcase` is by far the slowest.